content/post/2020-10-14-dv-signal.Rmd

```diff
@@ -0,0 +1,259 @@
+---
+title: "Using Electronic Health Records to Develop Real-Time COVID-19 Indicators"
+author: "Aaron Rumack"
+date: 2020-10-14
+tags: ["nowcasting", "data sources"]
+output:
+  html_document:
+    code_folding: hide
+---
+
+In addition to symptom surveys, one of our most prized data sources comes from health insurance claims data. Healthcare partners provide us with the counts of people going to the doctor due to COVID-like illness in counties across the entire country. We use this data to calculate a signal that estimates the level of COVID-like illness in every county where enough data is available. In this post, we discuss the data itself, what this data can tell us that the number of confirmed cases can't, and how we calculate a %CLI signal. We also investigate why the data appeared to become much less useful over time, and how we discovered that the data just needed a simple adjustment!
+
+<!--more-->
+
+In previous blog posts, we have written about indicators generated from surveys that we run through [Facebook](https://delphi.cmu.edu/blog/2020/08/26/covid-19-symptom-surveys-through-facebook/) and [Google](https://delphi.cmu.edu/blog/2020/09/18/covid-19-symptom-surveys-through-google/). In addition to these, we have also generated indicators from electronic medical records. We received access to hospital records and insurance claims data from healthcare partners covering 10-15% of the population of the United States. One indicator that we created from this data is the "Doctor Visits" signal, which estimates the percentage of office visits due to COVID-like illness.
+
+Electronic health records are extremely useful in providing information about epidemics in real time. A patient may interact with the healthcare system days before receiving a test (if any), and several more days before receiving a positive result and being reported as a confirmed case by a state's Department of Health. This can make our signal an **early indicator** of confirmed cases, and certainly an early indicator of deaths. Early knowledge of an increase in people with COVID-like symptoms is extremely important, as early containment can prevent an outbreak.
```

Comment:
```suggestion
Electronic health records are extremely useful in providing information about epidemics in near real time. A patient may interact with the healthcare system days before receiving a test (if any), and several more days before receiving a positive result and being reported as a confirmed case by a state's Department of Health. This can make our signal an **early indicator** of confirmed cases, and even more so an early indicator of deaths. Early knowledge of an increase in people with COVID-like symptoms is extremely important, as early containment can help contain an outbreak.
```